src/components/AudioCell.vue

```diff
@@ -433,6 +463,19 @@ export default defineComponent({
       textfield.value.focus();
     };
 
+    // 複数エンジン
+    const isMultipleEngine = computed(() => store.state.engineIds.length > 1);
+
+    const engineIcons = computed(
+      () =>
+        Object.fromEntries(
+          store.state.engineIds.map((engineId) => [
+            engineId,
+            store.state.engineInfos[engineId].iconData,
+          ])
+        ) as { [key: string]: string }
+    );
```

Comment:
`as { [key: string]: string }`はなくても良さそう？

```suggestion
    const engineIcons = computed(() =>
      Object.fromEntries(
        store.state.engineIds.map((engineId) => [
          engineId,
          store.state.engineInfos[engineId].iconData,
        ])
      )
    );
```

Reply:
消しましたー。